optaplanner-distribution/src/main/assembly/assembly-optaplanner.xml

```diff
@@ -77,6 +77,20 @@
         <exclude>.git/**</exclude>
       </excludes>
     </fileSet>
+    <fileSet><!-- Note: going outside the module dir is bad, but it is not fetching generated files -->
+      <directory>../optaplanner-quickstarts/quarkus-school-timetabling</directory>
+      <outputDirectory>quickstarts/quarkus-school-timetabling</outputDirectory>
```

Comment:
optaplanner-quarkus on 7.x is only build for community, not product (it's in a maven profile that depends on -Dproductized not being there).
So do not recommend adding quarkus-school-timetabling which depends on optaplanner-quarkus in the assembly, (unless it's somehow filtered out for product, but I'd keep it simple and just not add it at all).

Reply:
I guess there were some confusion about both quickstarts as @dupliaka  filed https://issues.redhat.com/browse/RHPAM-3265, we can close that jira and keep https://issues.redhat.com/browse/RHDM-1430 as that is for springboot alternative